packages/passport/sdk/src/zkEvm/transactionHelpers.ts

```diff
@@ -30,6 +30,13 @@ export type TransactionParams = {
   flow: Flow;
 };
 
+export type EjectionTransactionParams = Pick<TransactionParams, 'ethSigner' | 'zkEvmAddress' | 'flow'>;
+export type EjectionTransactionResponse = {
+  to: string;
+  data: string;
+  chainId: string;
+};
+
```

Comment:
If everything in this file is related to the ejection tx, any reason not to keep it inside `signEjectionTransaction.ts` ?